conans/test/util/tools_test.py

```diff
@@ -1167,3 +1171,47 @@ def build(self):
         client.save({"conanfile.py": conanfile, "other": "hello"})
         client.run("create . user/channel", ignore_error=True)
         self.assertIn("specify a branch to checkout", client.out)
+
+
+@attr('slow')
+class RealRequestTest(unittest.TestCase):
+    """
+    Open a real server to test tools that download stuff.
+    """
+
+    thread = None
+    file = None
+
+    def setUp(self):
+        if not self.thread:
+            # Create a tar file to be downloaded from server
+            with tools.chdir(tools.mkdir_tmp()):
+                import tarfile
+                tar_file = tarfile.open("sample.tar.gz", "w:gz")
```

Comment:
Probably simplify test, just download a .txt file

Reply:
Original issue was due to `tools.get()`. `tools.download` and `tools.unzip()` already have a `filename` parameter